src/components/charts/RiskList/index.js

```diff
@@ -45,7 +50,9 @@ const BoxChart = ({ chartActive, id, rtData }) => {
   return (
     <div className={classes.root}>
       <header className={classes.header}>
-        <Typography variant="h4">{CANADIAN_PROVINCES[id]}</Typography>
+        <Typography variant="h4">
+          {CANADIAN_PROVINCES[id] || ONTARIO_REGIONS[id]}
```

Comment:
@ryan-neufeld if you're interested, this one line is the reason that `NWT` was showing up as `Northwest Territories` instead of `Northwestern` for Ontario 🤦